tests/rmi.bats

```diff
@@ -87,6 +87,7 @@ load helpers
   run buildah --debug=false images -q
   [ $(wc -l <<< "$output") -eq 2 ]
 
+  buildah rmi --all --force
```

Comment:
Why run this twice?

Reply:
Now that we have the check for `isParent`, when we do `buildah rmi -a`, it is possible for it to hit an image that has children so in the first rmi -a run the children will be removed, then we can run it again to remove the remaining images.
This check is necessary so that users don't delete images that have dependent children.

Reply:
IDK, I think if a user does "rmi --all -force", they're not expecting to have to run it again to remove all, they just want them all gone in the first pass....

Reply:
I can add a check after to see if there are any images left, if so call removeImage() on them again. Does that sound better?

Reply:
Yes lets try that, but fail after two times.  Don't want to introduce an infinite loop